include/gz/sim/Link.hh

```diff
@@ -306,6 +306,19 @@ namespace gz
                                  const math::Vector3d &_force,
                                  const math::Vector3d &_torque) const;
 
+      /// \brief Add a wrench expressed in world coordinates and applied to
+      /// the link at the given offset from its center of mass. This wrench
+      /// is applied for one simulation step.
+      /// \param[in] _ecm Mutable Entity-component manager.
+      /// \param[in] _force Force to be applied expressed in world coordinates
+      /// \param[in] _offset The point of application of the force expressed
+      /// in the link-fixed frame and relative to the center of mass.
+      /// \param[in] _torque Torque to be applied expressed in world coordinates
+      public: void AddWorldWrench(EntityComponentManager &_ecm,
```

Comment:
The `AddWorldWrench` above applies the force at the link origin. In this overload, the offset is relative to the COM.  It might be a good idea to change the name to reflect that. Maybe `AddWorldWrenchRelativeToCOM`? /cc @scpeters.

Reply:
Renamed the function in 361cd4f